src/codeu/chat/UserMap.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import codeu.chat.client.simplegui.NewAccount;
+
+/**
+ * Created by Suveena on 5/24/17.
+ */
+
+/* Basic structure that holds all the users. Create this map upon starting GUI.
+Each time we add a new user, check to make sure that they don't already exist.
+If they do not, add them to the map.*/
```

Comment:
not sure if that's Troy for <curse word> or ...?

On the other hand, you can keep using a hashmap, it should just be on the server side and you'll access it via RPC